spec/models/outgoing_message_spec.rb

```diff
@@ -143,12 +143,36 @@
     expect(@outgoing_message.get_text_for_indexing).not_to include("foo@bar.com")
   end
 
-  it "should not display email addresses on page" do
-    expect(@outgoing_message.get_body_for_html_display).not_to include("foo@bar.com")
-  end
+  context "when formatting the message for html display" do
+
+    it "does not display email addresses on page" do
+      expect(@outgoing_message.get_body_for_html_display).not_to include("foo@bar.com")
+    end
+
+    it "links to help page where email address was" do
+      expect(@outgoing_message.get_body_for_html_display).to include('<a href="/help/officers#mobiles">')
+    end
+
+    it "does not force long lines to wrap" do
+      long_line = "123456789 123456789 123456789 123456789 123456789 123456789 1234"
+      @outgoing_message.body = long_line
+      expect(@outgoing_message.get_body_for_html_display).to include(long_line)
```

Comment:
This spec passes irrespective of whether the change to `app/models/outgoing_message.rb` is applied.

I can make the spec fail before the code change is applied by changing the spec to:

```diff
- @outgoing_message.body = long_line
+ @outgoing_message.body = long_line * 2
```

Since we're making a change here, could we make the example string a bit clearer? Something like `"String with more than 78 characters"` or something?

I find that writing specs first – or at least without the code change in place – usually catches this sort of thing.